src/sql/azdata.proposed.d.ts

```diff
@@ -4313,6 +4313,11 @@ declare module 'azdata' {
 			* @return A promise that resolves with a value indicating if the outputs are cleared or not.
 			*/
 			clearAllOutputs(): Thenable<boolean>;
+
+			/**
+			 * Changes the Notebook's kernel. Thenable will resolve only after kernel change is complete.
+			 */
+			changeKernel(kernel: IKernelSpec): Thenable<boolean>;
```

Comment:
Minor: do we get extra information back after changing a kernel? if so should we return that, or is the intent that folks will query the notebook session for that info instead?